roles/rac-lsnr-firewall/tasks/main.yml

```diff
@@ -45,3 +45,24 @@
   when: create_listener
   tags: lsnr-firewall
 
+- name: Allow local networks for RAC
+  firewalld:
```

Comment:
Firewalld won't work with EL6, however https://github.com/google/bms-toolkit/blob/master/docs/user-guide.md#target-server-requirements reports we only officially support 7.3+ anyway.  So this should be fine.

Reply:
Ack